data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -222,7 +222,7 @@
                                     [message]
                                         speaker=narrator
                                         image="wesnoth-icon.png"
-                                        message= _ "You struggle to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. You have increased resistance to all physical damage!"
+                                        message= _ "$unit.name| struggles to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. $unit.name| feels an increased resistance to all physical damage!"
```

Comment:
Well, in that case you could still do something like `$unit.name?The $unit.type|`.